volume/local/local.go

```diff
@@ -51,6 +52,7 @@ func New(scope string) (*Root, error) {
 	return r, nil
 }
 
+// Root holds volumes associated with a scope
```

Comment:
Root implements the Driver interface for the volume package and manages the creation/removal of volumes. It uses only standard vfs commands to create/remove dirs within its provided scope.